### DISCUSSION TITLE
Majora's Mask 5 AM bug

### DISCUSSION THREAD

**Discussion by n3t3n** [mupen64plus-ae/mupen64plus-ae#1175] [General]
Hello! I'm playing The Legend of Zelda: Majora's Mask (Japanese region) on mupen64 android (pro version). Whenever the game is about to reach 6 am (when a "new day" starts), it zooms in on the current time (the number "5"), instead of what it's supposed to do, make the screen smaller and smaller each time the clock rings. I've tried multiple resolutions (including the original resolution). Unfortunately, it changed nothing. Screenshot attached. Any help would be appreciated!
![Screenshot_20240405-012711_M64Plus FZ Pro](https://github.com/mupen64plus-ae/mupen64plus-ae/assets/16443663/1eb5bb2a-9b4b-4ed4-b03d-1822597b8744)

**Comment by andrewdinunzio** [comment]
Noticed the same. It's just like zoomed in to this texture but the game is still playing. I can move around, pause it (then the zoomed-in texture changes to some other text), etc. with the game music and sounds still working fine, I just can't see what I'm doing.

It's resolved when dawn of the next day starts.

![Screenshot_20250111-000519.png](https://github.com/user-attachments/assets/dca49b6e-8c0b-4044-aa53-1fc2b7019985)

**Comment by KITMORFO** [comment]
go to the emulation profile and change the video plug to "gliden64" and RSP "HLE".
